docs/examples/builds/multi-root-editor.md

```diff
@@ -14,180 +14,45 @@ The main difference between using a multi-root editor and using multiple separat
 
 {@snippet examples/multi-root-editor}
 
-## Editor example configuration
```

Comment:
While other "builds" will link to the builder, this one needs to stay I think as multi-level won't be there (yet).

Reply:
DO you mean we keep the sample code for this?

Reply:
Yes, sorry!